src/include/cluster/dialogs.rb

```diff
@@ -708,6 +708,205 @@ def SaveSecurity
       nil
     end
 
+    def ValidateCorosyncQdevice
+      if UI.QueryWidget(Id(:corosync_qdevice), :Value) == false
+        return true
+      end
+
+      if UI.QueryWidget(Id(:qdevice_model), :Value) != "net"
+        Popup.Message(_("The model basically defines the type of arbitrator, currently only net is support"))
+        UI.SetFocus(:qdevice_model)
+        return false
+      end
+
+      if UI.QueryWidget(Id(:qdevice_votes), :Value).to_i <= 0
+        Popup.Message(_("Qdevice votes must be a positive integer"))
+        UI.SetFocus(:qdevice_votes)
+        return false
+      end
+
+      if !IP.Check(UI.QueryWidget(Id(:qdevice_host), :Value))
+        Popup.Message(_("Qdevice host mush have a valid IP address"))
+        UI.SetFocus(:qdevice_host)
+        return false
+      end
+
+      if UI.QueryWidget(Id(:qdevice_port), :Value).to_i <= 0
+        Popup.Message(_("The corosync qdevice port must be a positive integer"))
+        UI.SetFocus(Id(:qdevice_port))
+        return false
+      end
+
+      if !["ffsplit", "lms", "test", "2nodelms"].include?(UI.QueryWidget(Id(:qdevice_algorithm), :Value))
+        Popup.Message(_("The algorithm only can be one of the ffsplit, lms, test or 2nodelms.
+          YaST will overwrite test and 2nodelms."))
```

Comment:
for longer lines it is better to not have garbage whitespaces in string so you can write it this way:
```
 Popup.Message(_("The algorithm only can be one of the ffsplit, lms, test or 2nodelms." \
￼   " YaST will overwrite test and 2nodelms."))
```

Reply:
Thanks. Remember it.